src/linker/Linker.Dataflow/DynamicallyAccessedMemberTypes.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Mono.Linker.Dataflow
```

Comment:
I think this should go into System.Diagnostics.CodeAnalysis namespace as well

Reply:
Won't we just get a conflict once the linker project is updated to .NET 5? At that point it will technically be redundant and could be deleted, but linker already carries its own copy of the `BindingFlags` enum instead of using the one from the platform... it would be nice to have a consistent approach.

Reply:
As the number of these framework copied files is growing it would be better to have a consistent approach and I'd suggest to place them under the same name/namespace to avoid confusing why they existing and if they are different to the original version. The code won't be redundant but we will just if-def it out for NET5. I'm fine to exclude BindingFlags as well. I think the original issue was that on some old versions of .NET Framework not all members were available.